tests_against_server/test_loops.py

```diff
@@ -197,6 +176,7 @@ def prog_maker(list_param):
 
     cfg = deepcopy(config)
     arange_param_list = [

```

Comment:
If you're trying to run the same test against multiple input values, you can use the `pytest.parameterize` decorator. This has the benefit of giving you better information if you ever start gathering statistics on your unit tests, and will also help you to find out which set of input parameters failed. It also allows you to not have to use a for-loop. It's not really essential here since these are server-tests, but it might be worth doing in the future if you don't already know about it

Reply:
Here is an example of the syntax in the pull request for `long_wait`:
```python
@pytest.mark.parametrize("wait_time", [4, 16, 100,
                                       dummy_max_wait_time-1,
                                       dummy_max_wait_time,
                                       dummy_max_wait_time+1,
                                       100*dummy_max_wait_time])
def test_long_wait_time_simulation(qmm, config, wait_time):
    """ Extracts the wait time from simulation and asserts it to be equal to the input wait time."""
    op = "playOp"

    with program() as prog:
        play(op, "qe1")
        long_wait(wait_time, threshold_for_looping=dummy_max_wait_time)
    ...
```

Reply:
Updated, thanks!